pdk/1.0/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -30,13 +30,13 @@ anypoint-cli-v4 pdk policy-project create --name <my-custom-policy>
 
 NOTE: If you receive a prompt requesting your group ID, see xref:policies-pdk-troubleshooting.adoc#group-id-error[Troubleshooting Group ID].
 
-Running the `create` command creates a custom policy project with the specified name. The project appears in a new directory or a directory specified using the `--output-dir` flag. The following tutorials refer to this as your project's root directory.
+Running the `make create` command creates a custom policy project with the specified name. The project appears in a new directory or a directory specified using the `--output-dir` flag. The following tutorials refer to this as your project's root directory.
```

Comment:
```suggestion
Running the `make create` command creates a custom policy project with the specified name. The project appears in a default directory or a directory specified using the `--output-dir` flag. The following tutorials refer to this as your project's root directory.
```
Aren't both of them new directories? I assume the first one refers to a default directory of some kind?